charts/parca/values.yaml

```diff
@@ -9,6 +9,10 @@ agent:
   # -- Allows disabling parca agent
   enabled: true
 
+  # -- parca agent config block
+  config:
+    relabel_configs:
```

Comment:
I checked that with empty relabel configs it captures all the pods. I did it like that because otherwise (if it it just empty) `helm-docs` (and I guess Helm in general) assumes it's a string value and for consistency with `server.config`, I wanted to make it a block.